tasks.py

```diff
@@ -0,0 +1,135 @@
+from pyspark.sql import SparkSession
+from datetime import datetime
+from pyspark.sql import Window
+from pyspark.sql import functions as f
+import Utilit as u
+
+
+
+def top_100(df):
+    """
+    Top 100 films of all years
+    """
+    return df \
+        .filter((f.col("numVotes") >= 100000) & (f.col("titleType") == "movie")) \
```

Comment:
you do this `filter` at least 2 times, for now, maybe better to do it once somewhere?